feature/session/src/main/res/values-ja/strings.xml

```diff
@@ -8,6 +8,7 @@
     <string name="session_detail_resource">資料</string>
     <string name="session_detail_resource_video">動画</string>
     <string name="session_detail_resource_slide">スライド</string>
+    <string name="session_detail_share_url">https://droidkaigi.jp/2019/timetable/%s</string>
```

Comment:
I don't think we need a translation for this 👍

Reply:
ah, actually maybe we do 👀 because this should transition to Japanese session

Reply:
Oh, i see. Thank you so much. :smile:

Reply:
Sorry @h3birth  :sob: I made a mistake here, could you add this URL back? because the URLs are different between English and Japanese, we should also have this URL in `values-ja` as well 🙇